verilog/tools/lint/verilog_lint.cc

```diff
@@ -92,7 +92,10 @@ ABSL_FLAG(
     "If true, print the description of every rule formatted for the "
     "Markdown and exit immediately. Intended for the output to be written "
     "to a snippet of Markdown.");
-
+ABSL_FLAG(
+    bool, print_rules_file, false,
+    "Print the default lint values in a format that can be used to create a"
```

Comment:
there is probably a space missing at the end of the string in this line, otherwise the `.rules.verible_lint` will attach right to it and look something like `"...create a.rules.vver..."`

Reply:
Nice pick up. Fixed.